packages/lambdas/src/fhir-to-medical-record.ts

```diff
@@ -52,132 +53,132 @@ const ossApi = apiClient(apiUrl);
 const cloudWatchUtils = new CloudWatchUtils(region, lambdaName, metricsNamespace);
 
 // Don't use Sentry's default error handler b/c we want to use our own and send more context-aware data
```

Comment:
Let's remove these comments now that you fixed this? Here and elsewhere please.

Reply:
done